config/deploy/production.rb

```diff
@@ -0,0 +1,71 @@
+# server-based syntax
+# ======================
+# Defines a single server with a list of roles and multiple properties.
+# You can define all roles on a single server, or split them:
+
+ip = '46.101.73.56'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.